.github/workflows/build-and-dockerize.yaml

```diff
@@ -1,6 +1,10 @@
 on:
   workflow_call:
     inputs:
+      # Reference to checkout
+      ref:
+        default: ''
+        type: string
       dockerize:
         default: true
         type: boolean
```

Comment:
I guess `publishLatest:` also needs to be configured externally using `inputs` when you use workflow call? otherwise it won't never tag as `lastest`